src/components/Footer.tsx

```diff
@@ -53,225 +54,226 @@ export interface LinkSection {
 export interface IProps {}
 
 const Footer: React.FC<IProps> = () => {
-  const intl = useIntl()
+  const { language } = useI18next()
+  const { t } = useTranslation()
 
-  const isPageRightToLeft = isLangRightToLeft(intl.locale as Lang)
+  const isPageRightToLeft = isLangRightToLeft(language as Lang)
 
   const [medBp] = useToken("breakpoints", ["md"])
 
   const linkSections: Array<LinkSection> = [
     {
-      title: "use-ethereum",
+      title: t("use-ethereum"),
```

Comment:
Do we want to apply this here?
Line 342 below passes this as a key to the Translation component:
```ts
            {linkSections.map((section: LinkSection, idx) => (
              <Box key={idx}>
                <Heading as="h3" fontSize="sm" lineHeight="1.6" my="1.14em">
                  <Translation id={section.title} />
                </Heading>
                <List fontSize="sm" lineHeight="1.6" fontWeight="400" m={0}>
                  {section.links.map((link, linkIdx) => (
```
@pettinarip Should we remove the use of the `<Translation/>` component below?

Reply:
Good catch. Yea, we don't need the `<Translation />` component here since we are already doing the translations above.

The reason I've changed it is because the scanner we are using can't detect translation keys from variables. In this case, the scanner will find this line `<Translation id={section.title} />` and it will not know what key `section.title` holds. Ideally, we should wrap all the translation key strings with the `t` function or the `Translation` component.